src/Tabs/TabTemplate.js

```diff
@@ -11,6 +11,7 @@ const TabTemplate = ({children, selected, style}) => {
   if (!selected) {
     templateStyle.height = 0;
     templateStyle.overflow = 'hidden';
+    templateStyle.display = 'none';
```

Comment:
`Added `display:none` to the non active tab
Tabs that had fixed content, will not hide the fixed content after changing to other tab`

Its actually a bug solved: tabs that has a position absolute div's inside it, will now hide the content when switching to other tab. When previously it didn't hide the absolute positioned div's